lib/catalogx_client/base_client.rb

```diff
@@ -30,21 +32,28 @@ def handle_request(url, http_verb, query_params: nil, body: nil)
         $statsd.count("catalogx_client.timeout.retry", 1)
         retry_count += 1
         if retry_count <= CatalogXClient.max_retry
-          sleep(1)
+          sleep(rand(0.8..1.5))
```

Comment:
the random value range is very small. I think it should be somewhere between 1 and 4 seconds.